locales/en.json

```
@@ -26,6 +26,7 @@
   "advice:table-presentation": "Layout table",
   "advice:no-href": "Link without target",
   "advice:no-https": "No HTTPS",
+  "advice:select-option-not-empty": "<select> should have empty [value] on first <option>",
```

We need to mention that it only applies to `required` select:
```suggestion
  "advice:select-option-not-empty": "<select required> should start with <option value="">",
```

And I'd go for code-ish content, since the docs are here to explain messages.

---

Done, only change I did was to escape the double quotes.